util/arena.cc

```diff
@@ -63,9 +65,15 @@ Arena::Arena(size_t block_size, size_t huge_page_size)
     hugetlb_size_ = ((kBlockSize - 1U) / hugetlb_size_ + 1U) * hugetlb_size_;
   }
 #endif
+  if (tracker_ != nullptr) {
+    tracker_->Allocate(kInlineSize);
+  }
 }
 
 Arena::~Arena() {
+  if (tracker_ != nullptr) {
+    tracker_->FreeMem();
```

Comment:
It shouldn't. Let me add an assert. I'm adding it here to avoid the memory counter to drift and never recover in case we have a bug somewhere.